conductor/blueprints/package/blueprint.py

```diff
@@ -35,8 +35,7 @@ def upload():
         abort(400)
     if jwt is None:
         abort(403)
-    callback = os_conductor+url_for('.callback')
```

Comment:
I think this means you can get rid of the `os_conductor` variable on L16, and the `OS_CONDUCTOR_URL` env var completely (and its mention in the README), as it's only there to help define the callback url.

Reply:
will do

Reply:
And `OS_API_URL` too :)

Reply:
done